.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+<!-- This comments are hidden when you submit the PR,
```

Comment:
`This comment is` or `These comments are`

Reply:
I can't make up my mind ;-)

Reply:
"These comments are ..."

Reply:
"All your comment are belong to us."